proto/messages.proto

```diff
@@ -53,11 +53,13 @@ message Text {
   repeated Mention mentions = 4;
   optional Quote quote = 5; // if this Text is part of a MessageEdit, this field is ignored
   optional bool expects_read_confirmation = 6 [default = false]; // whether the sender is expecting to receive a read confirmation
+  required bool legal_hold_activated = 7 [default = false]; // whether a user in the conversation is under legal hold
```

Comment:
Don't we want it to be optional? Old client will not send it and we want to be able distinguish that case from it being set to `false`.